packages/core/src/fhir-to-cda/cda-templates/commons.ts

```diff
@@ -575,3 +576,14 @@ export function buildAddressText(address: Address | undefined): string | undefin
   if (!address) return undefined;
   return `${address.line?.join(", ")}, ${address.city}, ${address.state} ${address.postalCode}`;
 }
+
+export function mapGenderCode(gender: string | undefined): CdaSexType {
+  switch (gender?.toLowerCase()) {
+    case "male":
+      return "M";
+    case "female":
+      return "F";
+    default:
+      return "UK";
+  }
```

Comment:
are we sure there isnt already a function in the codebase that does this? I feel like we have one of these definetly. It would be best if you used a common function since we need to support [this](https://github.com/metriport/metriport/issues/2253) asap and will need to do some refactoring across the codebase and so one less new function to update would be better

Reply:
Ya, I put this function under here - [link](https://github.com/metriport/metriport/blob/9d4dda78ad308fe2ca45329efe8473eed7c00b0c/packages/core/src/domain/patient-demographics.ts#L38-L39)